repose-aggregator/core/repose-core/src/main/java/org/openrepose/core/services/deploy/ContainerConfigurationListener.java

```diff
@@ -56,14 +56,11 @@ public synchronized void configurationUpdated(ContainerConfiguration configurati
                 if (ad.getCheckInterval() > 0) {
                     dirWatcher.updateCheckInterval(ad.getCheckInterval());
                 }
-
-                dirWatcher.updateArtifactDirectoryLocation(new File(ad.getValue()));
-
-                deploymentDirectory = new File(dd.getValue());
+                dirWatcher.updateArtifactDirectoryLocation(new File(ad.getValue().trim()));
+                deploymentDirectory = new File(dd.getValue().trim());
```

Comment:
Fair enough. Having clear documentation about the fully qualified path and handling of leading/trailing spaces would make me happy. Let me know once we've got those? I'll merge this PR then.